fn/client/src/components/checkout-table/Checkout-table.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import './Checkout-table.css';
+import { Table } from 'react-bootstrap'
+import { connect } from 'react-redux';
+
+
+const CheckoutTable = ({ products, currency }) => {
+
+    const [currencyIcon, setCurrencyIcon] = useState('$');
+
+    useEffect(() => {
+        currency === 1 ? setCurrencyIcon('€') : setCurrencyIcon('$');
+    }, [currency]);
+
+    const totallCounter = () => {
+        const row = products.map(product => product.price * product.quantity);
+        const total = row.reduce((sum, next) => sum + next)
+        return total
+    }
+
+    return (
+        <Table
+            bordered
+            hover
+            responsive>
+            <thead>
+                <tr>
+                    <th>Image</th>
+                    <th>Item name</th>
+                    <th>Quantity</th>
+                    <th>Item prise</th>
+                </tr>
+            </thead>
+            <tbody>
+                {products.map(productRow =>
+                    <tr key={productRow.id}>
+                        <td>
+                            <img
+                                className="order-tbl-img"
+                                alt="order-item"
+                                src={`/images/products/${productRow.images}`}
+                            />
+                        </td>
+                        <td>{productRow.title}</td>
+                        <td>{productRow.quantity}</td>
+                        <td>{`${Math.round(productRow.price * productRow.quantity * currency).toFixed(2)} ${currencyIcon}`}</td>
+                    </tr>
+                )}
+                <tr>
+                    <td colSpan="3">Total:</td>
+                    <td>{`${Math.round(totallCounter() * currency).toFixed(2)} ${currencyIcon}`}</td>
```

Comment:
Move all calculation to function ${Math.round(totallCounter() * currency).toFixed(2)} ${currencyIcon}
as 
const sumCalc = () => {
return Math.round(totallCounter() * currency).toFixed(2) + currencyIcon
}

Reply:
all calculations are moved to function